README.md

```diff
@@ -168,23 +168,30 @@ token of your choosing.
 Use the generated access token and your verify token to configure your bot:
 
 ##### ... pass a block, or
+
 ```ruby
 Facebook::Messenger.configure do |config|
   config.access_token = 'EAAG6WgW...'
+  config.app_secret = '__app_secret_here__'
   config.verify_token = 'my_voice_is_my_password_verify_me'
 end
 ```
 
 ##### ... set directly
+
 ```ruby
-Facebook::Messenger.configure.access_token = 'EAAG6WgW...'
-Facebook::Messenger.configure.verify_token = 'my_voice_is_my_password_verify_me'
+Facebook::Messenger.config.access_token = 'EAAG6WgW...'
+Facebook::Messenger.config.app_secret = '__app_secret_here__'
+Facebook::Messenger.config.verify_token = 'my_voice_is_my_password_verify_me'
 ```
 
 ##### ... then read directly
+
 ```ruby
 Facebook::Messenger.config.access_token
 # => EAAG6WgW...
+Facebook::Messenger.config.app_secret
+# => '__app_secret_here__'
 Facebook::Messenger.config.verify_token
 # => my_voice_is_my_password_verify_me
```

Comment:
I think we could remove all of this; consumers of the client shouldn't need to read back the configuration values.